webxr/Cargo.toml

```diff
@@ -35,7 +35,7 @@ openxr = { version = "0.19", optional = true }
 serde = { version = "1.0", optional = true }
 glow = "0.16"
 raw-window-handle = "0.6"
-surfman = { git = "https://github.com/servo/surfman", rev = "300789ddbda45c89e9165c31118bf1c4c07f89f6", features = [
+surfman = { git = "https://github.com/servo/surfman", rev = "e78ea1b24e1de57f24fc761186d8e9d9e43d660a", features = [
```

Comment:
Can we make a surfman release now?

Reply:
I was hoping to get this and land companion PR (this way we can still do revert if something goes wrong) then do release on weekend. See https://github.com/servo/surfman/issues/325#issuecomment-2590034982